week1/scary/kirikaza/.gitignore

```diff
@@ -0,0 +1,2 @@
+/*.hp
```

Comment:
Don't commit the `.gitignore` here, just add entries to the top-level `.gitignore`. Also is there any particular reason for gitignoring `.ps`?